Form-Controls/index.html

```diff
@@ -3,25 +3,78 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
+    <title>My Form Exercise</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
     <header>
       <h1>Product Pick</h1>
     </header>
+
     <main>
+      <h2>Order Form</h2>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <label for="name">Full name:</label>
+        <input
+          type="text"
+          id="name"
+          name="first_name"
+          placeholder="Enter full name"
+          autocomplete="name"
+          maxlength="32"
+          pattern="[A-Za-z ]{2,32}"
+          required
+          aria-describedby="nameHint"
+        />
+        <small id="nameHint">Only letters and spaces, 2–32 characters.</small>
+        <br /><br />
+        <label for="email">Email address:</label>
+        <input
+          type="email"
+          id="email"
+          name="user_email"
+          autocomplete="email"
+          placeholder="Enter your email"
+          pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
```

Comment:
This pattern looks good, but by using `type="email"` we don't need to provide a pattern. The browser will do this for us

[MDN Email Validation](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/email#validation)

We get "Basic Validation" out-the-box, and can use pattern for more advanced cases